bcda/api.go

```diff
@@ -16,18 +16,13 @@
      - application/fhir+json
      - application/json
 
-     Security:
-     - api_key:
-
      SecurityDefinitions:
-     api_key:
+     bearer_token:
           type: apiKey
-          name: Authorization
+          name: For bulkData endpoints. 1) Put your credentials in Basic Authentication, 2) get your token from /auth/token, 3) put your token here, after the word "Bearer"
```

Comment:
I think new lines would be jazzy if that is possible.
I also propose this:
For BulkData Endpoints require a Bearer Token (Do we have a userguide explanation? can we link it?).  
1) Put your credentials in Basic Authentication (is an anchor link possible?)
2) Request a bearer token from /auth/token
3) Put "Bearer {TOKEN}" in this field (no quotes) using the bearer token retrieved in step 2

Reply:
@knollfear 
- Best I can tell, newlines are not an option given that it's HTML as produced by YAML as processed through GoDoc.  We're stretching the `Name` field a bit far as it is!
- HTML tags also seem to be escaped, so no links.

I've adapted your suggestions to the text at the top of the page as well as the "Authorize" dialog:

<img width="767" alt="Screen Shot 2019-04-15 at 8 01 46 PM" src="https://user-images.githubusercontent.com/2533561/56173032-efe71c80-5fb9-11e9-90c1-4cbb0ef135fd.png">

<img width="520" alt="Screen Shot 2019-04-15 at 8 02 26 PM" src="https://user-images.githubusercontent.com/2533561/56173036-f4133a00-5fb9-11e9-9cd1-2dec3b89b91e.png">

Reply:
> * Best I can tell, newlines are not an option given that it's HTML as produced by YAML as processed through GoDoc.  We're stretching the `Name` field a bit far as it is!

Just 'cause I was curious, I went and looked and found this: <https://golang.org/pkg/go/doc/#ToHTML>. Which is dumb; not allowing for MarkDown or some more advanced embedded formatting than what they've got seems short-sighted. Oh well.

Reply:
@karlmdavis Thanks for that.  Yes, it all works as your reference says for the top of the page.  But in the middle of the `bearer_token` `Name:` field links aren't handled.  We'll keep that in mind for future enhancements.